examples/structured-keys/structured_keys.md

```diff
@@ -1,6 +1,13 @@
 # Structured keys
 
-Key fields can be structured as long as they belong to any subtype of plain data. The value of the key field must be immutable. The initializer of the `readonly` field will be constructed as immutable. In other cases, `cloneReadOnly()` can be used to create an immutable value.
+Key fields can be structured types as long as they belong to any subtype of plain data. Plain data is a simple value, a sequence value, or a structured value where all members are plain data.
+The value of the key field must be immutable.
+The initializer of the read-only field will be constructed as immutable. When using `add`/`put`, `cloneReadOnly()` can be used to create an immutable value.
```

Comment:
Shall we add the description in a single line? The contents in separate lines are rendered as separate paragraphs on the website.